src/components/DownloadArea.tsx

```diff
@@ -0,0 +1,53 @@
+import { CSVLink } from 'react-csv';
+import { RootState } from '@/app/store';
+import {
+  selectCountOfSelectedPeople,
+  selectSelectedPeople,
+  unselectAllPeople,
+} from '../features/people/peopleSlice';
+import { useDispatch, useSelector } from 'react-redux';
+
+export default function DownloadArea() {
+  const dispatch = useDispatch();
+  const selectedItemsCount = useSelector((state: RootState) =>
+    selectCountOfSelectedPeople(state)
+  );
+  const selectedItems = useSelector((state: RootState) =>
+    selectSelectedPeople(state)
+  );
+
+  if (selectedItemsCount <= 0) {
+    return null;
```

Comment:
can count be negative?
```suggestion
  if (!selectedItemsCount) {
    return null;
```